H/HiGHS/build_tarballs.jl

```diff
@@ -23,22 +23,33 @@ if [[ "${target}" == *86*-linux-musl* ]]; then
 fi
 mkdir -p HiGHS/build
 cd HiGHS/build
-apk add --upgrade cmake --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main
+apk add --upgrade cmake
 cmake -DCMAKE_INSTALL_PREFIX=$prefix \
     -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
     -DCMAKE_BUILD_TYPE=Release \
     -DBUILD_SHARED_LIBS=ON \
     -DFAST_BUILD=ON \
     -DJULIA=ON \
-    -DIPX=OFF ..
-cmake --build . --config Release --parallel
+    -DIPX=ON ..
+if [[ "${target}" == *-linux-* ]]; then
+        make -j $nproc
+else
+    if [[ "${target}" == *-mingw* ]]; then
+        cmake --build . --config Release
+    else
+        cmake --build . --config Release --parallel
+    fi
+fi
 make install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = expand_cxxstring_abis(supported_platforms())
 
+# Useful for testing to add the MIP and QP solvers.
+# filter!(Sys.iswindows, platforms)
```

Comment:
is this still necessary or was it just for local tests?

Reply:
The MIP and QP solver are coming in the next weeks so it may be necessary again until we clear the bugs. Hence the comment.